packages/contracts/test/contracts/OVM/execution/OVM_ExecutionManager/nuisance-gas.spec.ts

```diff
@@ -188,7 +188,7 @@ const test_nuisanceGas: TestDefinition = {
             // This is because there is natural gas consumption between the ovmCALL(GAS/2) and ovmCREATE, which allots nuisance gas via _getNuisanceGasLimit.
             // This means that the ovmCREATE exception, DOES consumes all nuisance gas allotted, but that allotment
             // is less than the full OVM_TX_GAS_LIMIT / 2 which is alloted to the parent ovmCALL.
-            nuisanceGasLeft: 4184829,
+            nuisanceGasLeft: 0x3f9e7f,
```

Comment:
Just curious, why hex here?

Reply:
This test is super brittle and needs to be updated frequently. The value we use to update this test comes in hex. I think people have been converting to decimal before changing the test, I figured it was easier to leave in hex since JavaScript understands it.